packages/core-components/src/icons/icons.tsx

```diff
@@ -27,7 +27,13 @@ function useSystemIcon(key: string, props: IconComponentProps) {
 }
 
 // Should match the list of overridable system icon keys in @backstage/core-app-api
-/** @public */
+/**
+ * Broken Image Icon from material UI library
```

Comment:
no need to mention what library it's from (can actually be any source)

Reply:
Ok, i had an idea to document our Icons in storybook and then add reference to it  :+1